modules/swiss/src/main/SwissApi.scala

```diff
@@ -385,6 +387,43 @@ final class SwissApi(
           }
     .void >> recomputeAndUpdateAll(id)
 
+  def playerReady(swissId: SwissId, userId: UserId): Funit =
```

Comment:
there's room for race conditions, causing a game to not start, or to start twice (with 2 game ids)